packages/nouns-webapp/src/config.ts

```diff
@@ -47,22 +47,22 @@ export const cacheKey = (bucket: CacheBucket, ...parts: (string | number)[]) =>
   return [bucket.name, bucket.version, ...parts].join('-').toLowerCase();
 };
 
-export const CHAIN_ID: SupportedChains = parseInt(process.env.REACT_APP_CHAIN_ID ?? '4');
+export const CHAIN_ID: SupportedChains = parseInt(import.meta.env.VITE_CHAIN_ID ?? '4');
 
-export const ETHERSCAN_API_KEY = process.env.REACT_APP_ETHERSCAN_API_KEY ?? '';
+export const ETHERSCAN_API_KEY = import.meta.env.VITE_ETHERSCAN_API_KEY ?? '';
 
 export const WALLET_CONNECT_V2_PROJECT_ID =
-  process.env.REACT_APP_WALLET_CONNECT_V2_PROJECT_ID ?? '';
+  import.meta.env.VITE_WALLET_CONNECT_V2_PROJECT_ID ?? '';
 
-const INFURA_PROJECT_ID = process.env.REACT_APP_INFURA_PROJECT_ID;
+const INFURA_PROJECT_ID = import.meta.env.VITE_INFURA_PROJECT_ID;
 
 export const createNetworkHttpUrl = (network: string): string => {
-  const custom = process.env[`REACT_APP_${network.toUpperCase()}_JSONRPC`];
+  const custom = import.meta.env[`VITE_${network.toUpperCase()}_JSONRPC`];
   return custom || `https://${network}.infura.io/v3/${INFURA_PROJECT_ID}`;
 };
 
 export const createNetworkWsUrl = (network: string): string => {
-  const custom = process.env[`REACT_APP_${network.toUpperCase()}_WSRPC`];
+  const custom = import.meta.env[`VITE_${network.toUpperCase()}_WSRPC`];
```

Comment:
something going on with these on Netlify
I have `VITE_MAINNET_WSRPC` and `VITE_MAINNET_JSONRPC` correctly set up, but it's falling back to the infura url

could the dynamic env be throwing off netlify?
I think we can get rid of this flaky logic and just do with `VITE_JSONRPC` and `VITE_WSRPC`

Reply:
I think it could pass by unnoticed locally if you had VITE_INFURA_PROJECT_ID set

Reply:
Makes sense, yeah; but at the same time, feels like a lot of this logic’s getting tossed anyway once #895 lands.

Reply:
fair enough. just hardcode those to `VITE_MAINNET_JSONRPC` and `VITE_MAINNET_WSRPC` for now so I can fix the Netlify build.

Testnet deployment is disabled anyway: #936

Reply:
pushed on 86028b5, but still not working.

found the likely culprit on build logs:
```sh
11:01:11 AM:   You need to set the output format to "esm" for "import.meta" to work correctly.
11:01:11 AM: ▲ [WARNING] "import.meta" is not available with the "cjs" output format and will be empty [empty-import-meta]
11:01:11 AM:     packages/nouns-webapp/src/config.ts:65:17:
11:01:11 AM:       65 │   const custom = import.meta.env.VITE_MAINNET_WSRPC;
```

not sure why it's assuming a cjs output

Reply:
nvm, those are from the `functions` build. still trying to figure out what's up

Reply:
Just remember to update `turbo.json` as well when you add or change ENV variables.